plugins/org.python.pydev/src/org/python/pydev/ui/pythonpathconf/package_manager/CondaPackageManager.java

```diff
@@ -35,6 +35,21 @@ public CondaPackageManager(IInterpreterInfo interpreterInfo, File prefix) {
         this.prefix = prefix;
     }
 
+    public static List<File> listCondaEnvironments(File condaExecutable) {
+        List<File> lst = new ArrayList<>();
+        String encoding = "utf-8"; // use system encoding
+        Tuple<String, String> output = new SimpleRunner().runAndGetOutput(
+                new String[] { condaExecutable.toString(), "env", "list", "--json" }, null, null,
+                null,
+                encoding);
+        JsonObject jsonOutput = JsonValue.readFrom(output.o1).asObject();
```

Comment:
I think we should `Log.logInfo(output.o1)` just in case we have an issue later on.

Reply:
Log it before `sonValue.readFrom` without surrounding it with try catch?

Reply:
Yes